ubuntu/debian/control

```diff
@@ -42,6 +44,7 @@ Depends: libignition-cmake2-dev,
          libignition-rendering5-dev,
          libignition-tools-dev,
          libignition-transport10-dev,
+         libignition-utils1-dev,
          libsdformat11-dev,
```

Comment:
Can we add cli too? See how we add `eigen3` for `ign-math`.

Reply:
Added in https://github.com/ignition-release/ign-edifice-release/pull/5/commits/f7bdd2e913e8522b9471d212106fa60d47b047d4. I wasn't sure if I should include this or not since the cli component is only a build dependency (see my [previous comment](https://github.com/ignition-release/ign-edifice-release/pull/5#issuecomment-799769197)). What's the difference between the `Build-Depends` and `Depends` sections of the control file?

Reply:
Ohh gotcha, sorry I missed the other comment. I can see the reasoning there. I don't think I fully grasp the difference, because I think that the runtime `ignition-edifice` package should probably not depend on `-dev` packages? Maybe @j-rivero can clarify :slightly_smiling_face:

Reply:
the `cli` component is only used for building the executables; it is not needed at runtime, so in other packages, I have made this a build dependency only (https://github.com/ignition-release/ign-transport10-release/pull/3)

Reply:
Thanks, @scpeters. I think that actually means that what I had originally (https://github.com/ignition-release/ign-edifice-release/pull/5/commits/e4d52817d3b527f316210f159a7568765f8e682f) was correct then, right? If so, I can revert https://github.com/ignition-release/ign-edifice-release/pull/5/commits/387ea3114c272d18b3579e1b4f2691fac0086c07.

Reply:
Yup let's just revert then. Thanks for iterating